tasks/es/cnig.py

```diff
@@ -152,6 +156,31 @@ def columns(self):
             )
         return cols
 
+class GeomNameColumns(ColumnsTask):
+
+    def version(self):
+        return 1
+
+    def requires(self):
+        return {
+                'geom_cols':GeometryColumns(),
+                'subsections':SubsectionTags(),
+                'sections':SectionTags(),
+            }
+
+    def columns(self):
+        cols = OrderedDict()
+        session = current_session()
+        for colname, coltarget in self.input()['geom_cols'].iteritems():
+            cols['name_' + colname] = OBSColumn(
+                type='Text',
```

Comment:
is that a problem for the GeomRefColumns task as well? I copied the code for that task over to this one, there weren't unique IDs for the GeomRefs columns there either...?